yarn.lock

```diff
@@ -3713,6 +3713,17 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@hyperlane-xyz/core@npm:1.0.0-beta3":
+  version: 1.0.0-beta3
+  resolution: "@hyperlane-xyz/core@npm:1.0.0-beta3"
+  dependencies:
+    "@hyperlane-xyz/utils": 1.0.0-beta3
+    "@openzeppelin/contracts": ^4.8.0
+    "@openzeppelin/contracts-upgradeable": ^4.8.0
+  checksum: b765ec01884fa897f89a9c64e3ff5c158e263b4ded6abc31faef3e45b500359836fd310be9b4bacdbb7ad23dc4706ae4fcd3f6abcb778c32db344e0d9b288f02
+  languageName: node
+  linkType: hard
+
```

Comment:
I wonder if these are from token and helloworld
I think this is fine moving forward but something to flag that they wont necessarily depend on the local version? I guess unless we add ^ rules

Reply:
Yeah I think it is from token + helloworld
I slightly disagree that it's fine. Seems important to keep them all in lockstep with monorepo itself uses helloworld, and will be difficult to keep the helloworld + token repos reguarly published otherwise.